docs/plugins/verifier/cosign.md

```diff
@@ -385,14 +403,19 @@ spec:
             {
                 "name":"cosign",
                 "artifactTypes": "application/vnd.dev.cosign.artifact.sig.v1+json",
-                "key": "/path/to/cosign.pub"
+                "rekorURL": "https://rekor.sigstore.dev"
             }
         ]
     }
 }
 ```
 
-### Usage
+Please note that the `key` is not specified in the config. This is because the keyless verification uses ephemeral keys and certificates, which are signed automatically by the [fulcio](https://github.com/sigstore/fulcio) root CA. Signatures are stored in the [Rekor](https://github.com/sigstore/rekor) transparency log, which automatically provides an attestation as to when the signature was created.
+
+The `rekorURL` MUST be provided for keyless verification. Otherwise, signature validation will fail.
```

Comment:
Why `rekorURL` is a MUST? If this parameter does not exist, the default Sigstore URL should be used, similiar to `cosign verify` command.

Reply:
Yeah great point. It ideally shouldn't be and that's why we introduced extra flag `tlogverify` to the new trust policy implementation. This is documenting the current legacy behavior which we will maintain for backwards compatability. This content in the legacy section is just rearranged which is why it's showing as a change.

Reply:
OK, this is under legacy section. Then it makes sense now.